olm-catalog/serverless-operator/Dockerfile

```diff
@@ -8,12 +8,12 @@ LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
 LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
 LABEL operators.operatorframework.io.bundle.package.v1=serverless-operator
 LABEL operators.operatorframework.io.bundle.channel.default.v1="stable"
-LABEL operators.operatorframework.io.bundle.channels.v1="stable,stable-1.29"
+LABEL operators.operatorframework.io.bundle.channels.v1="stable,stable-1.29,stable-1.30"
```

Comment:
do we want to append? (compare to @dsimansk's PR)

Reply:
Yes, I think it's OK and needed for the new channels we are introducing.